### PR TITLE
fix: building docker images from the compose

### LINKED ISSUE #19: [Bug] cant build docker-compose
**Describe the Bug**

```
docker compose up --build
```

fails

`apps/playwright-service` cannot be  found.
When substituting with `build: apps/puppeteer-service-ts`

however after this is fixed subsequently:

```
 CACHED [playwright-service stage-1  4/10] RUN npm install -g pnpm                                                                                                                                                                          0.0s
 => ERROR [playwright-service stage-1  5/10] COPY ./apps/puppeteer-service-ts/package.json ./                                                                                                                                                  0.0s
 => ERROR [playwright-service stage-1  6/10] COPY ./apps/puppeteer-service-ts/pnpm-lock.yaml ./                                                                                                                                                0.0s
 => CACHED [playwright-service stage-1  7/10] RUN pnpm install --frozen-lockfile                                                                                                                                                               0.0s
 => ERROR [playwright-service stage-1  8/10] COPY apps/puppeteer-service-ts .                                                                                                                                                                  0.0s
------
 > [playwright-service stage-1  5/10] COPY ./apps/puppeteer-service-ts/package.json ./:
------
------
 > [playwright-service stage-1  6/10] COPY ./apps/puppeteer-service-ts/pnpm-lock.yaml ./:
------
------
 > [playwright-service stage-1  8/10] COPY apps/puppeteer-service-ts .:
------
failed to solve: failed to compute cache key: failed to calculate checksum of ref 14d7568d-20b4-401d-ae2f-e5a06caa8490::xz1s7hiyuimwmw1l5thsraova: "/apps/puppeteer-service-ts": not found
```

fails

**To Reproduce**
Steps to reproduce the issue:
1. try to start docker compose up --build
2. fix the 1st issue
3. observe the 2nd issue

### PR DESCRIPTION
**Pull request by yudataguy**
Fixes Docker build context and resolves #19

This PR addresses two issues:
1. Resolves #19, add missing `playwright-service` folder, copied from firecrawl repo
2. Fixes the COPY error during `docker compose build` by aligning the Dockerfile path with the build context

The error was occurring because the Dockerfile and docker-compose build context were referencing different folder paths. This has been corrected and verified with successful curl tests.

Testing:
- Verified successful `docker compose build`
- Confirmed working with curl requests

### CODE REVIEW

**Comment by skeptrunedev** [comment]
@yudataguy this is closer to the right way of solving - https://github.com/devflowinc/firecrawl-simple/pull/21

TLDR `/apps/<path>` is what's messing it up. `playwright-service` was renamed to `puppeteer-service-ts` and is not missing. As mentioned in the README, we swapped out playwright for ulixee hero such that fire-engine is not required for private pages.

**Review by skeptrunedev — CHANGES_REQUESTED** [review]
- [x] remove `playwright-service` as `puppeteer-service` is a full replacement
- [ ] modify `/apps/api/Dockerfile` such that CI still passes **and** docker-compose build works
- [x] modify `/apps/puppeteer-service-ts/Dockerfile` such that CI still passes **and** docker-compose build works

**Comment by skeptrunedev** [comment]
> Not sure why the build failed, since the files are available, and local build succeed. Any idea?

I have ideas, but unconfirmed. It's erroring because "/src/lib/go-html-to-md": not found"  which is the old error the compose had. Changing the path in the dockerfile to no longer include `./apps` in [this pr](https://github.com/devflowinc/firecrawl-simple/pull/20/files#diff-9ada278a9adf2846ef744063b304ed43826d904a8e1729669a1035ee26e036c6R27) is probably what's messing it up. 

I suspect that the [context .](https://github.com/devflowinc/firecrawl-simple/blob/main/.github/workflows/build-docker-images.yml#L21) in the github CI workflow has to be changed to `./apps/api` and `./apps/puppeteer-service-ts`.

**Review by skeptrunedev — CHANGES_REQUESTED** [review]
- [x] bugfix CI context path

**Review by skeptrunedev — APPROVED** [review]
Cleaned up the commits and made some final fixes to the CI. Thank you for the contribution @yudataguy !
